tar_includes/README

```diff
@@ -1,17 +1,39 @@
 Thanks for using ungoogled-chromium! You have downloaded a Portable Linux build.
 
+SANDBOXING
 Before you run, you must setup your system so the browser's sandboxing will work. There are two options:
 * Use the user namespace sandbox. This is the preferred option.
 * Use the SUID sandbox. This should be used on systems that do not have support for user namespaces.
 
-How to setup the SUID sandbox:
+To quickly determine your current sandbox setup:
+
+1. Launch UC temporarily from a terminal:
+$ /opt/ungoogled-chromium_75.0.3770.80-1.1_linux/chrome
+2. Navigate to: chrome://sandbox/
+3. Your screen may declare, in part:
+Sandbox Status
+Layer 1 Sandbox Namespace
+PID namespaces Yes
```

Comment:
I don't know if these lines will change across Chromium versions, but it seems that the `You are adequately sandboxed.` line at the bottom always persists. Maybe this is a better indicator if any SUID sandbox stuff needs to be done?

Reply:
I will omit the guidance on navigating to "chrome://sandbox" in that case. 

In order to hopefully get further confirmation/guidance from coders about interpreting "chrome://sandbox" results before having this committed in the future, a new 'Enhancement' issue could be opened in this [ungoogled-chromium-portablelinux forum](https://github.com/ungoogled-software/ungoogled-chromium-portablelinux/issues).

Reply:
- A 'SANDBOXING' section title now appears, written in all caps.
- The original 'Running the browser:' section title now appears in caps to match and in order to promote this section title style from the 'How to set up the usernamespace/SUID sandbox' subsection titles.
- The 'How to set up the usernamespace sandbox' subsection now appears before SUID sandbox instructions, as namespace sandboxing is announced in the README file to be the preferred option (and I had moved the subsection first originally, as chrome://sandboxing apparently confirms that a user namespace sandbox was already active.)  
- In the 'How to set up the user namespace sandbox' subsection, the introductory sentence is prepended: "If there is no user namespace sandbox".